### ISSUE TITLE
[youtube] --dump-json "subtitles" key = null for video with subtitles

### ISSUE THREAD

**Issue by Vdragon** [rg3/youtube-dl#4602]
I'm writing a program to parse youtube video supported format and subtitles, however the `subtitles` key of JSON output of `youtube-dl --dump-json https://www.youtube.com/watch?v=iAy0tG56ZYg` is `null` while `youtube-dl --list-subs https://www.youtube.com/watch?v=iAy0tG56ZYg` declares that the video has zh-TW,en,ja subtitles:
```
[youtube] iAy0tG56ZYg: Downloading webpage
[youtube] iAy0tG56ZYg: Extracting video information
[youtube] iAy0tG56ZYg: Looking for automatic captions
[youtube] iAy0tG56ZYg: Downloading XML
WARNING: Video doesn't have automatic captions
[youtube] iAy0tG56ZYg: Available subtitles for video: zh-TW,en,ja
[youtube] iAy0tG56ZYg: Available automatic captions for video:
```

I wonder that if it's youtube-dl's bug for not listing the available subtitles in `subtitles` key, or if it isn't, how should I parse a video's available subtitles?

Tested using 2014.12.16.2 version of youtube-dl

**Comment by Vdragon** [comment]
@jaimeMF 
Thanks for replying, what I really need is a list of available subtitle languages(for a later youtube-dl call to actually download them, like `--list-subs`), however `--all-subs` includes entire subtitle content into the json which is quite more than I expected.

Is there any better way to accomplish the same goal?

**Comment by jaimeMF** [comment]
In the next version, `youtube-dl 'iAy0tG56ZYg' --all-subs --dump-json` will just return the subtitles urls and it won't download the contents with the `--dump-json` option (for reference: the changes are in PR #4964), therefore I'm closing the issue. Thanks for the report.

**Comment by phihag** [comment]
I believe this issue to be fixed in youtube-dl version 2015.02.24 and newer. See [our FAQ](https://yt-dl.org/update) if you need help updating.
